src/cli/command/run.go

```diff
@@ -204,7 +215,13 @@ func runApplication(c *cli.Context) error {
 			return errors.New("image can't be empty")
 		}
 
+		runas := c.String("run-as")
+		if runas == "" {
```

Comment:
can we inline helper methods like `func defaultStringValueIfEmpty(testValue, defaultValue string) defaultValue string `